packages/@react-types/shared/src/labelable.d.ts

```diff
@@ -40,5 +40,9 @@ export interface SpectrumLabelableProps extends LabelableProps {
   /**
    * Whether the label is labeling a required field or group.
    */
-  isRequired?: boolean
+  isRequired?: boolean,
+  /**
+   * A ContextualHelp element to place next to the label.
```

Comment:
Link to ContextualHelp? also, do we want to restrict the ReactNode to a ContextualHelp as well?

Reply:
probably not because then people couldn't wrap ContextualHelp into their own component